types/got/v8/tsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "strictFunctionTypes": true,
+        "baseUrl": "../../",
+        "typeRoots": [
+            "../../"
+        ],
+        "types": [],
+        "paths": {
+            "got": [
+                "got/v8"
+            ],
+            "got/*": [
```

Comment:
Could you explain why need this option? The `got` module has not such imports.

Reply:
I just copied it from another type definition I used as example.